### DISCUSSION TITLE
Anyway to speed up the intellisense when working with huge headers ?

### DISCUSSION THREAD

**Discussion by Mactarvish** [microsoft/vscode-cpptools#10068] [Q&A] [answered]
Intellisense of cpptools always works fine except when I work with too many headers (e.g. I included opencv, qt, boost together), under this situation the intellisense becomes very slow. Is there anyway to speed up the intellisense ? (e.g. increase the maximum number of intellisense could use, or increase the maximum memory it can use?)

**Comment by sean-mcmanus** [comment, marked as answer]
Making sure C_Cpp.intelliSenseCacheSize is large enough might help.
